gno.land/pkg/gnoland/types.go

```diff
@@ -53,6 +53,8 @@ func ReadGenesisTxs(ctx context.Context, path string) ([]TxWithMetadata, error)
 		scanner = bufio.NewScanner(file)
 	)
 
+	scanner.Buffer(make([]byte, 1_000_000), 2_000_000)
```

Comment:
For context on this -- the default scanner limit is ~65k chars, which proved to be too little for even our portal loop transactions, so I bumped the value to ~1M chars